studio-docs/source/data-privacy.md

```diff
@@ -31,6 +31,7 @@ All data sent to Apollo Studio is sent to an endpoint with one of the following
 | `https://operations.api.apollographql.com` | Apollo Server with the [operation registry plugin](https://www.apollographql.com/docs/studio/operation-registry/) (v0.4.1+) |
 | `https://storage-secrets.api.apollographql.com` | Apollo Server with Apollo Gateway (v0.16.0-v0.33.0) with [managed federation](https://www.apollographql.com/docs/federation/managed-federation/overview/), or with the [operation registry plugin](https://www.apollographql.com/docs/studio/operation-registry/) (v0.4.1+) |
 | `https://uplink.api.apollographql.com` | Apollo Server with Apollo Gateway (v0.34.0+) with [managed federation](https://www.apollographql.com/docs/federation/managed-federation/overview/) |
+| `https://aws.uplink.api.apollographql.com` | Apollo Server with Apollo Gateway (v0.34.0+) with [managed federation](https://www.apollographql.com/docs/federation/managed-federation/overview/) |
```

Comment:
Going to land this on the 0.45 release branch (after it lands on `main`), so let's go with that
```suggestion
| `https://aws.uplink.api.apollographql.com` | Apollo Server with Apollo Gateway (v0.45.0+) with [managed federation](https://www.apollographql.com/docs/federation/managed-federation/overview/) |
```